be/src/olap/types.h

```diff
@@ -601,22 +594,11 @@ struct FieldTypeTraits<OLAP_FIELD_TYPE_VARCHAR> : public FieldTypeTraits<OLAP_FI
         slice->size = value_len;
         return OLAP_SUCCESS;
     }
-    static void set_to_max(void* buf) {
```

Comment:
If you remove this function, it will use CHAR's set_to_max, which memset with slice->size. It is not what we need. I think you should set set_to_max to nullptr to avoid use VARCHAR's set_to_max. What's more I prefer that move CHAR's set_to_max to field too. In CharFiled we can use field's length other than slice->size, which is less error-prone.

Reply:
👌；We can unify char/varchar's set_to_max

Reply:
> set set_to_max to nullptr to avoid use VARCHAR's set_to_max
Here means set a Class's static function to nullptr?

Reply:
Yes, define a static function pointer who's value is nullptr;